exec/java-exec/src/main/java/org/apache/drill/exec/compile/ClassTransformer.java

```diff
@@ -240,7 +240,7 @@ public Class<?> getImplementationClass(
       Map<String, Pair<byte[], ClassNode>> classesToMerge = Maps.newHashMap();
       for (byte[] clazz : implementationClasses) {
         totalBytecodeSize += clazz.length;
-        final ClassNode node = AsmUtil.classFromBytes(clazz, ClassReader.EXPAND_FRAMES);
+        ClassNode node = AsmUtil.classFromBytes(clazz, ClassReader.SKIP_FRAMES);
```

Comment:
Does this change affect previous behavior?

Reply:
We use `ClassWriter#COMPUTE_FRAMES` for class writer, so according to the docs, `ClassReader.SKIP_FRAMES` is useful to avoid visiting frames that will be ignored and recomputed from the scratch. I had to set it in this PR because Janino has changed something in its bytecode, so it fails with the `ClassReader.EXPAND_FRAMES` flag.